docs/concepts.md

```diff
@@ -99,6 +133,33 @@ They currently include:
 - The initial (worker) node count
 - The Kubernetes version
 
-## Base cluster
+## Base Cluster 
+
+NOTE: The 'Base Cluster' is a new and evolved way of specifying cluster configuration. It will become the default mechanism for specifying cluster configuration starting version 0.10.0 of Arlon
+
+A Base Cluster allows you to create workload clusters from a manifest file stored in a git repository. The manifest
+typically contains multiple related resources that together define an arbitrarily complex cluster.
+If you make subsequent changes to the Base Cluster manifest, workload clusters originally created from it will automatically acquire the changes.
+The Base Cluster defines:
+
+- A predefined list of Cluster API objects: Cluster, Machines, Machine Deployments, etc. to be deployed in the current namespace
+- The specific infrastructure provider to be used (e.g aws)
+- Kubernetes version
+- Cluster nodepool type that need to be used for creating the cluster manifest (e.g eks, eks-managedmachinepool)
+
+To know more about 'Base Cluster', read about it [here](./baseclusters.md)
+
+## Cluster Chart
```

Comment:
The cluster chart is only used by gen1 clusters. Gen2 clusters created from base clusters don't use the cluster chart. It's confusing to describe both base clusters and cluster charts in the same document.

Reply:
okay then I will remove the cluster chart stuff from the 0.10 docs version, once I am able to see it on the doc site